projects/pgai/pgai/vectorizer/vectorizer.py

```diff
@@ -622,6 +789,67 @@ async def _embed_and_write(self, conn: AsyncConnection, items: list[SourceRow]):
 
         return len(records)
 
+    @tracer.wrap()
+    async def _embed_and_write_from_batch(
```

Comment:
Shouldn't this just be called `_write_from_batch`? Since the embedding has already completed when the function is called.

Reply:
Renamed it to `_write_embeddings_from_batch` - the original name was inspired by the same function which does this using the regular api.